README.md

```diff
@@ -68,6 +68,7 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
+      # if set to nil no timeout is used
```

Comment:
As a suggestion I find `nil` as an API not very intuitive. While it makes sense knowing internals can we maybe consider using a symbol like `: disabled` to be more declarative and also to avoid `nil` as it seems easier to debug getting `: disabled` no method error, pointing straight to where it comes from while `nil` no method error is much less explicit.